pkg/kf/buildpacks/client.go

```diff
@@ -107,7 +107,7 @@ func (c *client) fetchConfig(builderImage string) (*gcrv1.ConfigFile, error) {
 		return nil, err
 	}
 
-	image, err := c.imageFetcher(imageRef, remote.WithAuthFromKeychain(authn.DefaultKeychain))
+	image, err := c.imageFetcher(imageRef, remote.WithAuth(authn.Anonymous))
```

Comment:
I think its presumptuous to assume the builder file will always be public. The operator could always have a private builder for a space, and then this would no longer work.

Could we first try Anonymous, then fallback to the DefaultKeychain?

Reply:
I'm actually thinking we solve this differently, so lets go ahead and merge this.